lib/fluent/plugin/out_s3.rb

```diff
@@ -216,7 +216,12 @@ def write(chunk)
         tmp.rewind
         log.debug { "out_s3: write chunk: {key:#{chunk.key},tsuffix:#{tsuffix(chunk)}} to s3://#{@s3_bucket}/#{s3path}" }
 
-        put_options = {:body => tmp, :content_type => @compressor.content_type, :storage_class => @storage_class}
+        put_options = {
+          :body => tmp,
+          :content_type => @compressor.content_type,
+          :storage_class => @storage_class,
+          :acl => @acl
```

Comment:
[ACL overview](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl) describes ACL default is `private`.
But [SDK document](http://docs.aws.amazon.com/sdkforruby/api/Aws/S3/Object.html#put-instance_method) does not describes about default value of ACL.

BTW, in previous version S3 bucket and object is created as `private` ACL.